specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -1538,7 +1617,357 @@
           {
             "value": "en-us",
             "name": "EnUs",
-            "description": "English"
+            "description": "Query language value for en-us."
+          },
+          {
+            "value": "en-gb",
+            "name": "EnGb",
+            "description": "Query language value for en-gb."
+          },
+          {
+            "value": "en-in",
+            "name": "EnIn",
+            "description": "Query language value for en-in."
+          },
+          {
+            "value": "en-ca",
+            "name": "EnCa",
+            "description": "Query language value for en-ca."
+          },
+          {
+            "value": "en-au",
+            "name": "EnAu",
+            "description": "Query language value for en-au."
+          },
+          {
+            "value": "fr-fr",
+            "name": "FrFr",
+            "description": "Query language value for fr-fr."
+          },
+          {
+            "value": "fr-ca",
+            "name": "FrCa",
+            "description": "Query language value for fr-ca."
+          },
+          {
+            "value": "de-de",
+            "name": "DeDe",
+            "description": "Query language value for de-de."
+          },
+          {
+            "value": "es-es",
+            "name": "EsEs",
+            "description": "Query language value for es-es."
+          },
+          {
+            "value": "es-mx",
+            "name": "EsMx",
+            "description": "Query language value for es-mx."
+          },
+          {
+            "value": "zh-cn",
+            "name": "ZhCn",
+            "description": "Query language value for zh-cn."
+          },
+          {
+            "value": "zh-tw",
+            "name": "ZhTw",
+            "description": "Query language value for zh-tw."
+          },
+          {
+            "value": "pt-br",
+            "name": "PtBr",
+            "description": "Query language value for pt-br."
+          },
+          {
+            "value": "pt-pt",
+            "name": "PtPt",
+            "description": "Query language value for pt-pt."
+          },
+          {
+            "value": "it-it",
+            "name": "ItIt",
+            "description": "Query language value for it-it."
+          },
+          {
+            "value": "ja-jp",
+            "name": "JaJp",
```

Comment:
@shmed The name and description should be a lot more useful by actually mentioning the language. See LexicalAnalyzerName, SnowballTokenFilterLanguage, StemmerTokenFilterLanguage, etc.